web_notify/models/res_users.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
In python3 `# -*- coding: utf-8 -*-` it should be removed.

Same for the others python file in this pull request.

Reply:
It's not strictly needed, but optional when you have already encoded your file.